### PR TITLE
Include artifacts for all built projects directly in the linked page

### PR DESCRIPTION
**Pull request by tpoterba**
let's see what happens

### CODE REVIEW

**Comment by cseed** [comment]
Looks like the artifacts didn't copy.

**Comment by tpoterba** [comment]
yeah, how do I do this? We need test the new CI, build it, and use the new CI to build the rest.

**Comment by tpoterba** [comment]
@danking 	do you have advice?

**Comment by danking** [comment]
The currently running CI is the old one, it chooses the index file. If you want to test that the index file looks the way you want, you'd need to add a `ci/` test. It's not super straightforward, but `ci/test-in-cluster.sh` starts a CI pointing at a test repo. This eventually calls `ci/test/test-ci.py` which triggers builds and deploys on the test repo. Those builds and deploys should have the expected index file.

File: ci/index.html [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
+
+<h1>Hail</h1>
+<object type="text/html" data="artifacts/hail.html"></object>
+<hr>
+
+<h1>Batch</h1>
+<object type="text/html" data="artifacts/batch.html"></object>
+<hr>
+
+<h1>Batch</h1>
```
Comment (danking) [inline ci/index.html]: h1s are wrong for this one and ones after

**Comment by danking** [comment]
IMHO, I'd just merge it if you're happy with everything else. Breaking the build results page only affects developers and it doesn't even really break anything since we can use `gsutil` to directly access the results.

File: ci/index.html (hunk 1 above)
Comment (tpoterba) [reply ci/index.html]: thanks

**Comment by tpoterba** [comment]
I'm quite certain this will break the CI artifacts, but I'll fix it ASAP when we merge (so maybe let's hold off until tomorrow)

**Comment by danking** [comment]
bump, this is in my queue right now.

**Comment by tpoterba** [comment]
I need to stage this. change CI index to support more files, then make hail changes.

**Pull request closed without merging**